src/main/external-resources/renderers/DefaultRenderer.conf

```diff
@@ -618,7 +621,8 @@ CharMap =
 #    qdmc     QDesign Music Codec (RaveSound)
 #    ralf     RealAudio Lossless Format                     audio/vnd.rn-realaudio
 #    rm       RealMedia (RMVB)                              application/vnd.rn-realmedia-vbr
-#    shn      Shorten
+#    shn      Shorten                                       audio/x-shorten
+#    sipro    Sipro Lab Telecom
```

Comment:
Should be added:
`#    sls     Scalable to Lossless                       audio/mp4                          Also known as MPEG-4 SLS`